ui/src/translations/en.json

```diff
@@ -84,5 +84,8 @@
   "no": "No",
   "volume_creation": "Volume creation",
   "volume_creation_success": "Volume {name} has been created successfully.",
-  "volume_creation_failed": "Volume {name} creation has failed."
+  "volume_creation_failed": "Volume {name} creation has failed.",
+  "volume_name_error": "Name must be lowercase letters, numbers and hyphens.",
```

Comment:
Not totally accurate (`you can use `.`, you can't use `-` at the start or end, …) but good enough I think.
An accurate message would be too long for an error message (probably OK for tooltip though)